src/Components/dashboard/route/AddRoutes.jsx

```diff
@@ -80,7 +80,9 @@ const AddRoutes = () => {
     if (stop === "") {
       setStopError(true);
     } else {
+
       console.log(routeID, startPoint, endPoint, stop);
```

Comment:
Same here let's remove these console log lines

Reply:
Thank you for the comment @abraham-kwizera , I have resolved this